portal-web/test/functional/com/liferay/portalweb/macros/FormsAdminNavigator.macro

```diff
@@ -93,6 +93,12 @@ definition {
 			value1 = "${formName}");
 	}
 
+	macro gotoForm{
```

Comment:
Could you please check if this macro is doing a similar function as we have in `FormsAdminNavigator.gotoEditFormByName`?
If they're both editing a form while receive its name as a parameter, may I ask you to remove those lines and use the `gotoEditFormByName` instead?

Reply:
It does. Removed.